test_extendable_pydantic_fastapi/routers.py

```diff
@@ -21,11 +21,22 @@ def post_user(user: User) -> UserSearchResponse:
 
 
 @demo_pydantic_router.post("/post_private_user")
-def post_private_user(user: PrivateUser) -> UserSearchResponse:
+def post_private_user(user: PrivateUser) -> User:
     """A demo endpoint to test the extendable pydantic model integration
     with fastapi and odoo.
 
     Type of the request body is PrivateUser. This model inherits base model
-    User.
+    User but does not extend it.
```

Comment:
@marielejeune We should explain that for security, a method will always return attributes from declared response type. It should never return return attribute of a derived type from the declared response type even if an instance of the method return an instance of the derived type. The next test ensure that this assertion is also true with generics.